src/menu.hpp

```diff
@@ -283,6 +283,8 @@ extern Sint32 oldXres;
 extern Sint32 oldYres;
 extern button_t* revertResolutionButton;
 
+typedef std::tuple<int, int> resolution;
```

Comment:
I like it! Named datatypes sure make the code so much more reasonable! ("Mommy, why is he passing in _standard tuple int int_")